docs/providers/spark.md

```diff
@@ -0,0 +1,81 @@
+# Spark
+
+Featureform supports [Spark on AWS](https://aws.amazon.com/emr/features/spark/) as an Offline Store
+
+## Implementation <a href="#implementation" id="implementation"></a>
+Featureform implements spark as a transformation layer. The results of the transformations and table creations are stored in S3.
+
+Using the client, a user can define transformtions using dataframes, which transform into new tables. The function name and decorator variant label can be used as an identifier to further chain transformations, and use the new tables to create training sets.
+
+Current support requires an [AWS EMR](https://aws.amazon.com/emr/) cluster and an [S3](https://aws.amazon.com/s3/) bucket to store the featureform generated tables, feature definitions and training sets.
+Features can be copied from spark to an inference store (ex: Redis)(link to page) for real-time feature serving
+
+Data is stored in parquet tables in the bucket. The spark store executes a spark job on your cluster to transform the data and copy it to its relevant sources for training and serving
```

Comment:
Confused about what you mean by "copy it to its relevant sources for training and serving"